### DISCUSSION TITLE
Video Output

### DISCUSSION THREAD

**Discussion by GeekyGhost** [deforum-art/sd-webui-text2video#92] [Ideas]
I've been using this and it has a lot of potential. I had issues with the video stitching initially and then I wasn't overly a fan of the final stitch, so I used my  Concactenator! (Concatenator... lol) to stitch the videos together. If any of this code is helpful or useful have at it. I was originally just using it for my content creation to stitch a folder of images together, but it's grown a bit beyond that at this stage, I've been working on incorporating it into image generation. Feel free to use any thing that may be useful and thanks for your project! 

https://github.com/GeekyGhost/Concactenator/blob/Concactenator/concactenator.py

this wasn't supposed to be feet lol, but that's what it ended up being lol. Your script generated the images, but the resulting video was sporadic, so I used my script to set a lower fps and stitch them together. I don't know if it's helpful or not, but this was just an attempt to show appreciation for your work lol. Sorry!

https://user-images.githubusercontent.com/111990299/229847744-b7be1f1f-e902-452a-b161-395f40cee0b1.mp4

**Comment by hithereai** [comment]
FFmpeg is being used in the repo, and you can control FPS, CRF and preset using the output tab. 
Appreciate the intentions, but you just made simple things much, much more complicated. :-)

![image](https://user-images.githubusercontent.com/121192995/229860345-4671d0f1-0f3b-474c-8501-55305d7bfb8a.png)

**Reply by GeekyGhost** [reply]
It was throwing errors and couldn't make the video, when it did even with the same fps it was sporadic. Sorry, I don't tend to post the errors I get as I just work around them usually or try to fix them on my end and forget about it. In any case, the only videos I'm able to get are ones where the build in one breaks. When it converts to video it's usually a black screen or a bit like the examples on the readme. No worries, like I said, may not be helpful. Just appreciate the work you put in and attempted to show appreciation by sharing what I had to use to make it work. 

when the built in functions convert the images to video it tends to be like this below or it's jittery and weird. 
So, I just let it fail to make the video and then convert them separately. 
https://user-images.githubusercontent.com/111990299/229896271-23f66905-63bd-473a-af6e-d0b9b80a9626.mp4

the image generation portion is great. Stays really cohesive and transitions well.

**Reply by GeekyGhost** [reply]
I guess for clarification because I'm often misunderstood, if the script were functioning correctly I wouldn't have made the post, but I don't really need help, I have a work around, I was sharing my work around in the hopes if someone else was having issues you may have more data on how to resolve it. Again, thanks for your work. It wasn't a complaint, was literally an attempt at a thank you.

**Reply by hithereai** [reply]
If you do some hacks to change the image to video stitching, just share a post about it specifically. 

If your code is just combining the images into a video, I don't see how it will yield a "better" video than ffmpeg.

**Reply by GeekyGhost** [reply]
Fair enough, thanks for the feedback and I will keep it in mind for the future.
